cmake/templates/python_distutils_install.bat.in

```diff
@@ -7,7 +7,9 @@ if DEFINED DESTDIR (
 
 cd "@INSTALL_CMD_WORKING_DIRECTORY@"
 
-mkdir "@SETUPTOOLS_INSTALL_PREFIX@\@PYTHON_INSTALL_DIR@"
+if NOT EXIST "@SETUPTOOLS_INSTALL_PREFIX@\@PYTHON_INSTALL_DIR@\" (
```

Comment:
The backslash only on the closing quote but not on the opening one looks weird. Are you sure about this?

Reply:
The backslash wasn't for character escaping and it is part of the string. Ending with the path separator will make the batch script to check the directory existence, and only if the directory does exist, it skips the directory creation.